crates/biome_cli/tests/snapshots/main_commands_migrate_eslint/migrate_eslintrcjson_donot_override_existing_config.snap

```diff
@@ -0,0 +1,33 @@
+---
+source: crates/biome_cli/tests/snap_test.rs
+expression: content
+---
+## `biome.json`
+
+```json
+{ "linter": { "rules": { "suspicious": { "noDoubleEquals": "error" } } } }
+```
+
+## `.eslintrc.json`
+
+```json
+{ "rules": { "eqeqeq": "off" } }
+```
+
+# Emitted Messages
+
+```block
+biome.json migrate ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  i Configuration file can be updated.
+  
+    1   │ - {·"linter":·{·"rules":·{·"suspicious":·{·"noDoubleEquals":·"error"·}·}·}·}
+      1 │ + {·"linter":·{·"rules":·{·"suspicious":·{·"noDoubleEquals":·"error"·}·}·}·}
+      2 │ + 
```

Comment:
If the only change to be done is adding a newline, I think it makes more sense to say there are no updates to be done. Otherwise this is just a fancy way of invoking the formatter :)

Reply:
Unfortunately, the formatter is always invoked before writing the file.